VKontakte/VKontakte/CustomView/ABCustomView.swift

```diff
@@ -0,0 +1,90 @@
+// ABCustomView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+@IBDesignable final class ABCustomView: UIView {
```

Comment:
```suggestion
@IBDesignable final class ABCustomView: UIView {
```
AB - не пишем префиксы мы не в обж с
Custom - никогда не пишем дай норм название